app/gui2/src/stores/graph/index.ts

```diff
@@ -176,6 +176,13 @@ export const useGraphStore = defineStore('graph', () => {
     return mod.insertNewNode(mod.doc.contents.length, ident, expression, meta)
   }
 
+  // Create a node from a source expression, and insert it into the graph.
```

Comment:
Not obvious why the return type is optional.

Reply:
Added a comment.